composer.json

```diff
@@ -20,24 +20,27 @@
     "config": {
         "sort-packages": true,
         "platform": {
-            "php": "7.3.99"
+            "php": "7.4.99"
+        },
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
         }
     },
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0"
+        "php": "^7.4 || ~8.0.0 || ~8.1.0"
     },
     "require-dev": {
-        "container-interop/container-interop": "^1.1",
+        "psr/container": "^1.1.2 || ^2.0.2",
```

Comment:
shouldn't this be a requirement rather than a dev requirement?
The code is actively typing against and since these PSR standards are only interfaces, requiring them would be fine?